crates/infra/cli/src/commands/publish/lock_files/mod.rs

```diff
@@ -0,0 +1,62 @@
+use std::path::Path;
+
+use anyhow::Result;
+use infra_utils::commands::Command;
+use infra_utils::git::TemporaryChangeset;
+use infra_utils::github::GitHub;
+use infra_utils::paths::PathExtensions;
+
+use crate::commands::publish::DryRun;
+
+pub fn publish_lock_files(dry_run: DryRun) -> Result<()> {
+    let head_branch = "infra/update-lock-files";
+
+    let mut changeset = TemporaryChangeset::new(head_branch, "update lock files after release")?;
+
+    Command::new("npm")
+        .arg("install")
+        .flag("--package-lock-only")
+        .run()?;
+
+    let local_changes = Command::new("git")
+        .arg("status")
+        .flag("--short")
+        .evaluate()?
+        .trim()
+        .to_owned();
+
+    if local_changes.is_empty() {
+        println!("No changes to lock files.");
+        return Ok(());
+    }
+
+    changeset.expect_change(Path::repo_path("package-lock.json"));
```

Comment:
Possibly dumb question: Would it be possible to already include the changed `package-lock.json` as part of the PR that updates the versions? I understand that we manually release the packages in the topological order and so this seems possible at a first glance.

I'd prefer if we removed the 2-step process and do everything in a lockstep to streamline the infra and the process for everyone (if it's possible).

Reply:
Not at all. But in the current setup, it is not possible to combine them unfortunately :(

- The PR that updates versions happens **before** we publish. The act of merging it starts the publish process. The changes there are source changes (our own configs, changelogs, etc...).
- The lock files PR created here happens **after** we publish, because it needs to pull the "integrity" hashes from NPM for the already published packages.

Reply:
Actually, the additional PR won't be required if we merge #957
I updated this PR!

Reply:
Ah, okay, I missed the details about the `integrity` hashes. I'm happy we will have a more traditional/streamlined dependency setup as the optional platform deps one is a bit wonky. i.e. this now passes and doesn't require the 2-step update.

@OmarTawfik thanks for looking into this!